src/test/java/io/xstefank/wildfly/bot/RuleMissingIdTest.java

```diff
@@ -0,0 +1,32 @@
+package io.xstefank.wildfly.bot;
+
+import io.quarkiverse.githubapp.testing.GitHubAppTest;
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.GHEvent;
+
+import java.io.IOException;
+
+import static io.quarkiverse.githubapp.testing.GitHubAppTesting.given;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+
+@QuarkusTest
+@GitHubAppTest
+public class RuleMissingIdTest {
+
+    @Test
+    public void testMissingRuleId() throws IOException {
+        given().github(mocks -> mocks.configFileFromString(
+                "wildfly-bot.yml", """
+                    wildfly:
+                      rules:
+                        - title: "Test"
+                          notify: [7125767235,0979986727]
+                    """))
+        .when().payloadFromClasspath("/pr-opened.json")
+        .event(GHEvent.PULL_REQUEST)
+        .then().github(mocks -> {
+            verifyNoMoreInteractions(mocks.ghObjects());
```

Comment:
you might want to try to catch the message log - https://github.com/smallrye/smallrye-health/blob/4c79455b2e78c42a9a745cbabbf7e67c5fde4fd6/implementation/src/test/java/io/smallrye/health/SmallRyeHealthReporterTest.java#L44